pelorus-operator/helm-charts/pelorus/subcharts/exporters/templates/_imagestream.yaml

```diff
@@ -6,5 +6,5 @@ metadata:
   labels:
     app.kubernetes.io/name: {{ .app_name }}
     app: {{ .app_name }}
-  name: {{ .app_name }}
+  name: {{ .app_name }}:
```

Comment:
this isn't missing `{{ .image_tag | default "v2.0.7" }}` from `charts/pelorus/subcharts/exporters/templates/_imagestream.yaml`?